Dockerfile

```diff
@@ -1,44 +1,42 @@
 ARG BASE=11.5.0-devel-ubuntu20.04
 FROM nvidia/cuda:${BASE}
 
-LABEL maintainer="J.C.Knight@sussex.ac.uk"
-LABEL version="4.8.0"
```

Comment:
Do you know if there's any way to read this from the version.txt? I've managed to convince setuptools and doxygen to do this but couldn't figure it out for docker

Reply:
Well passing it as a `build-arg` is simple and explicit but I expect you could also do it with the right `cat` / `sed` incantation.